extide/libs.gradle/nbproject/project.properties

```diff
@@ -21,5 +21,6 @@ javac.compilerargs=-Xlint -Xlint:-serial
 
 # For more information, please see http://wiki.netbeans.org/SignatureTest
 sigtest.gen.fail.on.error=false
+sigtest.public.packages=org.gradle
```

Comment:
This is wrong - and masks the real problem: sigtest fails to parse the class files. IMHO we should in this case disable sigtest and not pretend it does something sensible here. A comment documenting the problem might help future readers.